app/build.gradle

```diff
@@ -63,4 +63,6 @@ dependencies {
     implementation "com.google.dagger:dagger-android:$rootProject.dagger"
     kapt "com.google.dagger:dagger-android-processor:$rootProject.dagger"
     kapt "com.google.dagger:dagger-compiler:$rootProject.dagger"
+
+    testImplementation 'org.mockito:mockito-inline:2.8.47'
```

Comment:
We should use MockK for our mocking, instead of Mockito.

See: https://mockk.io

Reply:
I'd like to hear an explanation for this in person. Seems nitpicky.

Reply:
Mockito has a number of issues with Kotlin interoperability, such as the fact that `when` is a reserved keyword in Kotlin.

Beyond that, MockK uses a very functional approach, which is more idiomatic.

It leverages many language features of Kotlin that Mockito, even Mockito-Kotlin, simply don't do.